model/generate.go

```diff
@@ -282,14 +282,16 @@ func (g *GeneratedProject) saveNewBuildsAndTasks(ctx context.Context, v *Version
 }
 
 type specificActivationInfo struct {
-	tasks    map[string][]string // tasks by variant that have batchtime or activate specified
-	variants []string            // variants that have batchtime or activate specified
+	stepbackTasks map[string][]string
+	tasks         map[string][]string // tasks by variant that have batchtime or activate specified
```

Comment:
Is there something we can add to the name "tasks" to make it more specific and more easily distinguishable from stepbackTasks?

Reply:
yeah fair nit, I couldn't think of anything good but besides "activateTasks" and "activateVariants" but that's probably better than nothing